src/group-constructors.js

```diff
@@ -19,7 +19,52 @@
   }
 
   function constructor(children, timingInput) {
+    if (this._parent === undefined)
+      this._parent = null;
     this.children = children || [];
+    for (c in this.children) {
+      this.children[c]._parent = this;
+    }
+    this._isAncestor = function(effect) {
+      var a = this._parent;
+      if (effect == this)
+        return true;
```

Comment:
Super nit: you can just start with `var a = this;` and then get rid of this if statement.